packages/fxa-payments-server/src/routes/Subscriptions/index.stories.tsx

```diff
@@ -165,19 +165,20 @@ type SubscriptionsRouteProps = {
 };
 
 // Quick & dirty utility to help build variant stories with common route props
-const subscriptionsRouteWithBaseProps = (
-  baseRouteProps?: Partial<SubscriptionsProps>
-) => ({ routeProps = baseProps, ...otherProps }: SubscriptionsRouteProps) => (
-  <SubscriptionsRoute
-    {...{
-      routeProps: {
-        ...routeProps,
-        ...(baseRouteProps || {}),
-      },
-      ...otherProps,
-    }}
-  />
-);
+const subscriptionsRouteWithBaseProps =
+  (baseRouteProps?: Partial<SubscriptionsProps>) =>
+  ({ routeProps = baseProps, ...otherProps }: SubscriptionsRouteProps) =>
+    (
+      <SubscriptionsRoute
+        {...{
+          routeProps: {
+            ...routeProps,
+            ...(baseRouteProps || {}),
+          },
+          ...otherProps,
+        }}
+      />
+    );
```

Comment:
Prettier changes -- a previous version of this commit had modified this file (find-and-replace).